### PR TITLE
round-2

### PR DESCRIPTION
**Pull request by AliIbadullayev**
Changed way of rounding to fixed length a double

### CODE REVIEW

**Comment by ryukzak** [comment]
Thank you! Can you extract this function to misc module and use in all such places?

**Comment by AliIbadullayev** [comment]
* Rewrote test in modern way  according to #15
* Added 'to-fixed' func in misc module and replaced in all places where used  
* Removed unused functions and unused modules from headers 
* Added some other function to misc module: `quiz-answ`

**Comment by ryukzak** [comment]
Sorry, but can you split it into several PRs by their purpose? Contact me by telegram.

File: src/course_bot/misc.clj [hunk 1]
```diff
@@ -36,3 +36,5 @@
 
 (defn normalize-time [dt]
   (str-time (read-time dt)))
+
+(defn to-fixed [num len] (double (/ (Math/round (* (Math/pow 10 len) num)) (Math/pow 10 len))))
```
Comment (ryukzak) [inline src/course_bot/misc.clj:40]: Please, rename it to round-2

**Review by ryukzak — CHANGES_REQUESTED** [review]
Also, can you squash all you commits in one.

File: src/course_bot/misc.clj (hunk 1 above)
Comment (AliIbadullayev) [reply src/course_bot/misc.clj]: done

**Pull request merged**